ValueError: Buffer dtype mismatch, expected 'float64_t' but got 'float'

When reading a float32 file in Python library, we get the following error:
```
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
bindings/mscompress.pyx:815: in mscompress.Spectrum.peaks.__get__
    return np.column_stack((self.mz, self.intensity))
bindings/mscompress.pyx:804: in mscompress.Spectrum.mz.__get__
    self._mz = self._file.get_mz_binary(self.index)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   mz_array = np.asarray(<np.float32_t[:count]>float_ptr)
E   ValueError: Buffer dtype mismatch, expected 'float64_t' but got 'float'

```

--- a/python/bindings/mscompress.pyx
+++ b/python/bindings/mscompress.pyx
@@ -552,6 +552,7 @@ cdef class BaseFile:
     cdef Spectra _spectra
     cdef RuntimeArguments _arguments
     cdef z_stream* _z
+    cdef int output_fd
 
 
     def __init__(self, bytes path, size_t filesize, int fd):
@@ -562,6 +563,7 @@ cdef class BaseFile:
         self._spectra = None
         self._arguments = RuntimeArguments()
         self._z = _alloc_z_stream()
+        self.output_fd = -1
 
 
     def __enter__(self):

--- a/python/pyproject.toml
+++ b/python/pyproject.toml
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "mscompress"
-version = "1.0.1a3"
+version = "1.0.1a4"
 description = "Python bindings for mscompress, a compression library for mass spectrometry data"
 readme = "README.md"
 requires-python = ">=3.9"

--- a/python/test/test_mscompress.py
+++ b/python/test/test_mscompress.py
@@ -75,6 +75,14 @@ def test_mzml_context_manager(mzml_file):
         assert f.filesize == os.path.getsize(mzml_file)
 
 
+@pytest.mark.parametrize("msz_file", test_msz_data)
+def test_msz_context_manager(msz_file):
+    with read(msz_file) as f:
+        assert isinstance(f, MSZFile)
+        assert f.path == os.path.abspath(msz_file).encode('utf-8')
+        assert f.filesize == os.path.getsize(msz_file)
+
+
 @pytest.mark.parametrize("mzml_file", test_mzml_data)
 def test_describe_mzml(mzml_file):
     mzml = read(mzml_file)

